emacs-travis.mk

```diff
@@ -108,11 +108,14 @@ install_cask:
 
 install_texinfo:
 	@echo "Install Texinfo $(TEXINFO_VERSION)"
-	@curl -o "/tmp/texinfo-$(TEXINFO_VERSION).tar.gz" \
+	@curl -sS -o "/tmp/texinfo-$(TEXINFO_VERSION).tar.gz" \
 		'http://ftp.gnu.org/gnu/texinfo/texinfo-$(TEXINFO_VERSION).tar.gz'
 	@tar xzf "/tmp/texinfo-$(TEXINFO_VERSION).tar.gz" -C /tmp
 	@cd "/tmp/texinfo-$(TEXINFO_VERSION)" && \
 		./configure --quiet --enable-silent-rules --prefix="$(HOME)" $(SILENT)
```

Comment:
The `sed` call works, but couldn't you add `CFLAGS=-Wno-unused-result` to this configure call rather?